>From 58bab7e2f213300e1a5a51807f4fbbd1cf8a9cf9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Bri=C3=A8re?= <fbriere@fbriere.net>
Date: Tue, 30 Sep 2014 15:42:38 -0400
Subject: [PATCH] Strip leading <tab>s from the here-document in logrotate
 config

A here-document delimiter cannot normally have leading spaces; the shell
will consider the "EOF" line to be part of the input, and feed it to
chronyc (resulting in an "Unrecognized command" error).
---
 debian/logrotate | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/logrotate b/debian/logrotate
index 68f4518..aac3492 100644
--- a/debian/logrotate
+++ b/debian/logrotate
@@ -9,7 +9,7 @@
 	create 644
 	postrotate
 		PASSWORD=`awk '$1 ~ /^1$/ {print $2; exit}' /etc/chrony/chrony.keys`
-		cat << EOF | /usr/bin/chronyc | sed '/^200 OK$/d'
+		cat <<- EOF | /usr/bin/chronyc | sed '/^200 OK$/d'
 		password $PASSWORD
 		cyclelogs
 		EOF
-- 
2.1.1

